lib/diagnostic.js

```diff
@@ -28,7 +39,11 @@
 // Attributes: name, address, numberOfPets
 
 // Replace null with your answer
-let owner = null;
+let owner = {
+  name: 'Caleb',
+  address: '1 Some Street',
+  numberofPets: 5,
+};
 
```

Comment:
Hi @BenGitsCode - so is the mistake here just an extra comma on the last property?